db/model/aspect.js

```diff
@@ -214,58 +208,6 @@ module.exports = function aspect(seq, dataTypes) {
         );
       }, // hooks.beforeDestroy
 
-      /**
-       * Recalculate sample status if any of the status ranges changed.
-       * If aspect tags changed, send a "delete" realtime event for all the
-       * samples for this aspect. (The afterUpdate hook will subsequently send
-       * an "add" event.) This way, perspectives which filter by aspect tags
-       * will get the right samples.
-       *
-       * @param {Aspect} inst - The instance being updated
-       * @returns {Promise}
-       */
-      beforeUpdate(inst /* , opts */) {
-        const statusRangeWasUpdated = inst.changed('criticalRange') ||
-          inst.changed('warningRange') ||
-          inst.changed('infoRange') ||
-          inst.changed('okRange');
-        if (statusRangeWasUpdated) {
-          return new seq.Promise((resolve, reject) =>
-            inst.getSamples()
-            .each((samp) => {
-              samp.aspect = inst;
-              samp.calculateStatus();
-              samp.setStatusChangedAt();
-              samp.save();
-            })
-            .then(() => resolve(inst))
-            .catch((err) => reject(err))
-          );
-        }
-
-        if (inst.changed('tags')) {
-          return new seq.Promise((resolve, reject) => {
-            inst.getSamples()
-            .each((samp) =>
-              common.sampleAspectAndSubjectArePublished(seq, samp)
-              .then((published) => {
-                if (published) {
-                  return common.augmentSampleWithSubjectAspectInfo(seq, samp)
-                  .then((s) => {
-                    common.publishChange(s, sampleEventNames.del);
-                  });
-                }
-
-                return seq.Promise.resolve(true);
-              }))
-            .then(() => resolve(inst))
-            .catch(reject);
-          });
-        } // tags changed
-
-        return seq.Promise.resolve(true);
-      }, // hooks.beforeUpdate
-
```

Comment:
I am creating a few stories to handle this the right way.

Reply:
Were we not doing this at all with the sample store enabled?

Reply:
No, we were not.